.github/workflows/python-app.yml

```diff
@@ -54,21 +52,38 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest tox cairo-lang
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install flake8
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Test with tox and pytest
+
+  coverage:
+    runs-on: ubuntu-latest
+    if: ${{ always() }}
+    needs: [version, linter]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
```

Comment:
any reason to pick 3.8 and not 3.9? do we know if there's a performance difference for example? it'd be relevant if we're back to sequential testing

Reply:
Not sure if there are performance differences, I will check tomorrow (basically, I kept the version that was being used in the test job)